src/ndpp_header.F90

```diff
@@ -0,0 +1,109 @@
+module ndpp_header
+
+  use constants, only: ZERO
+
+  implicit none
+
+!===============================================================================
+! GRPTRANSFER contains probability tables for the unresolved resonance range.
+!===============================================================================
+
+  type GrpTransfer
+    real(8), allocatable :: outgoing(:,:) ! Outgoing transfer probabilities
+                                          ! Dimension of (moments, gmin:gmax)
+  end type GrpTransfer
```

Comment:
I'd prefer the ever-so-slightly more verbose but less ambiguous `GroupTransfer`.